OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
I assume you mean `Targetable` (unchanged), `TargetableFootprint` (moving `ITargetablePositions` from Building to this), and so on?

Unless I'm missing something, that still doesn't solve the RA/TD problem: for weapon range etc., the closest TargetablePosition needs to be used, but the projectiles must ignore that and fly to the CenterPosition.

I might be overlooking something, but as far as I can tell, we need that `TargetAcquisition` switch to avoid balance regressions in the legacy mods, I see no way around that.